covid_moonshot/analysis/plots.py

```diff
@@ -44,19 +44,19 @@ def plot_work_distribution(
     )
 
     distplot(
-        forward_works,
+        np.array(forward_works) * KT_KCALMOL,
```

Comment:
Does `KT_KCALMOL` mean "convert kT to kcal/mol"? Might want to add the "to" in future to make this clear.

Reply:
It's supposed to mean something like "kT in kcal/mol". Maybe `KT_IN_KCALMOL` or `KT_PER_KCALMOL` would be a better name.

Reply:
Eventually it seems like it would be nice to banish "bare" numbers in most of the code and use `value_in_unit` at the site where the value is needed. But I'm new to this dimensionful style and not sure if that's totally feasible.